### PR TITLE
Prettier 2.0: Latest @looker/prettier-config & eslint-config

### PR DESCRIPTION
**Pull request by lukelooker**
### :sparkles: Changes

- Added `lint:prettier` task to verify all files are passing
- Added .prettierignore to keep prettier out of build artifacts
- Applied prettier to all files in repository
- Lots of minor changes to support new Prettier 2.0 defaults

### :white_check_mark: Requirements

- [x] Includes test coverage for all changes
- [x] Build and tests are passing
- [ ] PR is ideally < 400LOC

### CODE REVIEW

File: packages/components/src/ActionList/ActionList.tsx [hunk 1]
```diff
@@ -119,17 +119,17 @@ export const ActionListLayout: FC<ActionListProps> = ({
 export const ActionList = styled(ActionListLayout)<ActionListProps>`
   ${ActionListRowColumns} {
     display: grid;
-    grid-template-columns: ${props =>
-      props.columns.map(column => `${column.widthPercent}%`).join(' ')};
+    grid-template-columns: ${(props) =>
+      props.columns.map((column) => `${column.widthPercent}%`).join(' ')};
```
Comment (dbchristopher) [inline packages/components/src/ActionList/ActionList.tsx]: @lukelooker so we do want parens everywhere now? We should probably revert that last change I made to eslint-config 🤦‍♂

Comment (dbchristopher) [reply packages/components/src/ActionList/ActionList.tsx]: Seeing these conflicting rules play out on save in vscode -- 
save-conflict

Comment (dbchristopher) [reply packages/components/src/ActionList/ActionList.tsx]: Yeah, looks like they changed the default for 2.0

Probably the easiest thing at this point is to revert to the old style, unless there is some practical reason why parentheses are better in general

Screen Shot 2020-03-31 at 10 13 50 AM

**Comment by dbchristopher** [comment]
@lukelooker looks like there is also a conflicting rule with trailing commas. I didn't initially notice this also changed on save: 
Screen Shot 2020-03-31 at 10 25 24 AM

File: packages/components/src/ActionList/ActionList.tsx (hunk 1 above)
Comment (lukelooker) [reply packages/components/src/ActionList/ActionList.tsx]: Yeah - I already did so :)

Comment (lukelooker) [reply packages/components/src/ActionList/ActionList.tsx]: https://github.com/looker-open-source/eslint-config/commit/43b139df4efed4cfd98c199f36e73437cb1d2715

Comment (dbchristopher) [reply packages/components/src/ActionList/ActionList.tsx:123]: oh! so is that just an issue with my VScode? 🤔

**Pull request merged**